test/spec/modules/adotBidAdapter_spec.js

```diff
@@ -0,0 +1,3179 @@
+import { expect } from 'chai';
+import { executeRenderer } from 'src/Renderer';
+import * as utils from 'src/utils';
+import { spec } from 'modules/adotBidAdapter';
```

Comment:
Please include the `.js` extension on these imports as well (eg. `src/Renderer.js`).